bftengine/src/bftengine/messages/MsgsCertificate.hpp

```diff
@@ -166,7 +175,7 @@ void MsgsCertificate<T, SelfTrust, SelfIsRequired, KeepAllMsgs, ExternalFunc>::a
 
     if (!ExternalFunc::equivalent(representativeMsg, msg)) return;  // msg should be ignored
 
-    relevantClass = 0;
+    relevantClass = 0;  // the class of the self message
```

Comment:
@oded282 can you explain what this "class" means? it's not very obvious...
Thanks!

Reply:
Sure.